src/cli/cli_main.py

```diff
@@ -196,7 +210,8 @@ def rm_data(name: str) -> None:
 
     # remove dataset from registry
     pz.DataDirectory().rmRegisteredDataset(name)
-
+# Notify the user that the dataset has been deleted
```

Comment:
See above - these types of comment do not really increase readability of the code.